docs/openapi.yml

```diff
@@ -219,20 +219,87 @@ paths:
 
   /v1/table/{name}/insert/:
     post:
-      description: Insert data to a table
+      description: Insert new data to the Table.
       tags:
         - Data
       operationId: insertData
-      summary: Insert data to a table
+      summary: Insert new data.
       parameters:
         - $ref: "#/components/parameters/table_name"
       requestBody:
         $ref: "#/components/requestBodies/arrow_stream_buffer"
-
       responses:
         "200":
           description: Insert successful
-
+        "400":
+          $ref: "#/components/responses/invalid_request"
+        "401":
+          $ref: "#/components/responses/unauthorized"
+        "404":
+          $ref: "#/components/responses/not_found"
+  /v1/table/{name}/merge_insert/:
+    post:
+      description: Create a "merge insert" operation
+        This operation can add rows, update rows, and remove rows all in a single
+        transaction. See python method `lancedb.table.Table.merge_insert` for examples.
+      tags:
+        - Data
+      summary: Merge Insert
+      operationId: mergeInsert
+      parameters:
+        - $ref: "#/components/parameters/table_name"
+        - name: on
+          in: query
+          description: |
+            The column to use as the primary key for the merge operation.
+          required: false
```

Comment:
I believe this is required (or should be)

Reply:
Fixed.